raster/src/main/scala/geotrellis/raster/io/geotiff/reader/TiffTagsReader.scala

```diff
@@ -106,7 +113,7 @@ object TiffTagsReader {
         byteBuffer.readDoublesTag(tiffTags, tagMetadata)
     }
 
-  implicit class ByteBufferTagReaderWrapper(val byteBuffer: ByteBuffer) extends AnyVal {
+  implicit class ByteReaderTagReaderWrapper(val byteBuffer: ByteReader) extends AnyVal {
```

Comment:
Oh nice, compile-time unboxing with type safety. I've seen an analogue to that in other languages but didn't know Scala could do it.